Source/SelfService/Web/package.json

```
@@ -27,6 +27,7 @@
         "@mui/lab": "5.0.0-alpha.78",
         "@mui/material": "5.6.2",
         "@mui/styles": "5.6.2",
+        "@mui/x-data-grid-pro": "^5.15.0",
```

We try to use absolute versions - to avoid strange NPM upgrades without us trying to do it:
```suggestion
        "@mui/x-data-grid-pro": "5.15.0",
```